_posts/2020-10-09-from-sprockets-to-webpacker.markdown

```diff
@@ -0,0 +1,315 @@
+---
+layout: post
+title: "From Sprockets to Webpacker"
```

Comment:
Just a suggestion:

```suggestion
title: "How to migrate from Sprockets to Webpacker"
```

Reply:
Maybe that sounds like we are migrating all assets from sprockets to webpacker (even CSS, images, etc)? We are only moving the JS.

Reply:
Maybe `How to Migrate your JavaScript from Sprockets to Webpacker`? maybe it's too long

Reply:
@arielj sounds good 👍 

```suggestion
title: "How to Migrate your JavaScript from Sprockets to Webpacker"
```